src/views/News/index.jsx

```diff
@@ -49,18 +51,23 @@ const StudentNews = () => {
   const [currentlyEditing, setCurrentlyEditing] = useState(false); // false if not editing, newsID if editing
   const cropperRef = useRef();
   const isAuthenticated = useIsAuthenticated();
+  const isAdmin = userIsInAuthGroup('NewsAdmin');
 
   const loadNews = useCallback(async () => {
     setLoading(true);
     if (isAuthenticated) {
       const newsCategories = await newsService.getCategories();
       const personalUnapprovedNews = await newsService.getPersonalUnapproved();
+      if (isAdmin) {
+        const unapprovedNews = await newsService.getUnapproved();
+      }
```

Comment:
```suggestion
      const unapprovedNews = isAdmin ? await newsService.getUnapproved() : null;
```
This is a little bit cleaner _in my opinion_ and it means that `unapprovedNews` will be explicitly set to null if the user is not an admin.